packages/api/src/schema/api-schema.yaml

```diff
@@ -1033,9 +1033,53 @@ components:
       properties:
         name:
           type: string
+          description: Display name
+          example: name
         canPublish:
           type: boolean
           description: Whether a user is allowed to publish audio/video tracks
+          example: true
+    create-room-response:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The ID of the room
+          example: d32ae9e6-c459-4931-9898-e86e2f5e7e16
+    room-user-response:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The ID of the user
+          example: d32ae9e6-c459-4931-9898-e86e2f5e7e16
+        joinUrl:
+          type: string
+          description: Joining URL
+          example: https://meet.livepeer.chat
+        token:
+          type: string
+          description: Joining JWT
+          example: token
```

Comment:
Does join URL already contain JWT? Might be worth being more explanatory in the description and explain how they can be used.